packages/zendesk-adapter/src/change_validators/one_translation_per_locale.ts

```diff
@@ -0,0 +1,106 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import _ from 'lodash'
+import {
+  ChangeValidator,
+  getChangeData, InstanceElement,
+  isAdditionOrModificationChange, isInstanceElement, ReferenceExpression,
+} from '@salto-io/adapter-api'
+import Joi from 'joi'
+import { createSchemeGuardForInstance, getParents } from '@salto-io/adapter-utils'
+// eslint-disable-next-line no-restricted-imports
+import { findDuplicates } from '@salto-io/lowerdash/dist/src/collections/array'
```

Comment:
we should not add restricted imports - see [this](https://github.com/salto-io/salto/blob/main/packages/adapter-utils/src/element_source.ts#L19-L22) for how this can be imported correctly
```suggestion
// eslint-disable-next-line no-restricted-imports
import { findDuplicates } from '@salto-io/lowerdash/dist/src/collections/array'
```